src/repror/internals/commands.py

```diff
@@ -8,32 +8,51 @@
 from pathlib import Path
 from subprocess import CompletedProcess
 import io
+from enum import Enum
 
 
 def run_command(command, cwd=None, env=None, silent=False) -> CompletedProcess:
     """Run a specific command."""
     return subprocess.run(command, cwd=cwd, env=env, check=True, capture_output=silent)
 
 
-def run_streaming_command_stderr(
+class StreamType(Enum):
+    """Enum to specify the type of stream to capture."""
+    STDOUT = 1
+    STDERR = 2
+
```

Comment:
let's add 

@property
def is_stderr:
   return self == StreamType.STDOUT

Reply:
Changed it!